src/modules/user-open-orders/selectors/user-open-orders.js

```diff
@@ -11,17 +11,32 @@ import { CLOSE_DIALOG_CLOSING } from 'modules/market/constants/close-dialog-stat
 
 import { formatNone, formatEtherTokens, formatShares } from 'utils/format-number'
 
+export function selectAllUserOpenOrderMarkets() {
+  const { loginAccount, orderBooks } = store.getState()
```

Comment:
This is an older pattern that actually is still present in a few locations, but shouldn’t be implemented going forward.

Take a look at how we’ve structured selectors with the `createSelector` convention and refactor for congruence to that structure.  

Also, break out this method into a separate file.

Reply:
fixed to use createSelector pattern.